CONTRIBUTING.md

```diff
@@ -38,6 +38,15 @@ This is a rough outline of what a contributor's workflow looks like:
 - Push your changes to a topic branch in your fork of the repository.
 - Make sure the tests pass, and add any new tests as appropriate.
 - Submit a pull request to the original repository.
+- Submit a comment with the sole content "@reviewer PTAL" (please take a look) in GitHub
+  and replace "@reviewer" with the correct recipient.
+  Do not reassign the pull request.
```

Comment:
can you just omit this line? I'm not convinced it warrants calling out, sometimes it can be useful